App.tsx

```diff
@@ -1,51 +1,26 @@
 /**
  * Sample React Native App
  * https://github.com/facebook/react-native
- * 
+ *
  * Generated with the TypeScript template
  * https://github.com/emin93/react-native-template-typescript
- * 
+ *
  * @format
  */
 
 import React, {Component} from 'react';
-import {Platform, StyleSheet, Text, View} from 'react-native';
-
-const instructions = Platform.select({
-  ios: 'Press Cmd+R to reload,\n' + 'Cmd+D or shake for dev menu',
-  android:
-    'Double tap R on your keyboard to reload,\n' +
-    'Shake or press menu button for dev menu',
-});
+import {View} from 'react-native';
+import Counter from './Counter';
+import styles from './styles';
 
 interface Props {}
-export default class App extends Component<Props> {
+interface State {}
+export default class App extends Component<Props, State> {
   render() {
     return (
       <View style={styles.container}>
-        <Text style={styles.welcome}>Welcome to React Native, Jon!</Text>
-        <Text style={styles.instructions}>To get started, edit App.tsx</Text>
-        <Text style={styles.instructions}>{instructions}</Text>
+        <Counter />
       </View>
     );
   }
 }
-
-const styles = StyleSheet.create({
```

Comment:
Keep styles with the components they style. This makes it so it's easier to see which styles affect which components.